tests/test_docker_images.py

```diff
@@ -0,0 +1,39 @@
+import docker
+from tests.helm_template_generator import render_chart
+import jmespath
+import pytest
+
+
+def list_docker_images():
+    extra_globals = {
+        "global.baseDomain": "foo.com",
+        "blackboxExporterEnabled": True,
+        "postgresqlEnabled": True,
+        "prometheusPostgresExporterEnabled": True,
+        "pspEnabled": True,
+        "veleroEnabled": True,
+    }
+
+    public_repo_docs = render_chart(values={"global": extra_globals})
+
+    # Listing docker images
+    search_string = "spec.template.spec.containers[*].image"
+    docker_images = []
+    for public_repo_doc in public_repo_docs:
+        docker_image_sublist = jmespath.search(search_string, public_repo_doc)
+
+        if docker_image_sublist is not None:
+            docker_images = docker_image_sublist + docker_images
+
+    return docker_images
+
+
+# Pulling docker image test
+@pytest.mark.parametrize("docker_image", list_docker_images())
+def test_docker_image(docker_client, docker_image):
+    docker_image = docker_image.replace('"', "").strip()
+    try:
+        # Pulling docker image
+        docker_client.images.pull(docker_image)
```

Comment:
@mishah334 it looks like this pulls the image. it would be awesome if we could find a way to validate the image without having to pull it. pulling all of these images will add a ton of network consumption, so it would be great to avoid it if possible.